src/utils.ts

```diff
@@ -243,6 +247,11 @@ export function copyTemplate(inDir: string, outDir: string, { _opts, ...vars }:
       if (removeFiles.length > 0) {
         await del([...removeFiles]);
       }
+      createdFiles = await Promise.all(createdFiles.map(async (file) => {
+        const changedFileName = file.replace('.template', '');
+        await moveFile(file, changedFileName);
+        return changedFileName;
+      }));
```

Comment:
@jgab-net What is this mainly for?

Reply:
this is to move the templates generated with the .template extension, for example style.scss.template to style.scss

Reply:
apparently copy-template-dir can't do it, so I move them, I could filter the array to move only those that have .template, what do you think?

Reply:
@jgab-net I think it needs to be judged, only the files with the suffix `.template` need to be moved.

Reply:
ready, I made the change